app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorViewController.kt

```diff
@@ -129,4 +142,63 @@ class SyncIndicatorViewController @Inject constructor(
         }
         .flatMap { syncStream().mergeWith(errorsStream()) }
   }
+
+  private fun isSyncPending(events: Observable<UiEvent>): Observable<UiChange> {
```

Comment:
This function is named like it returns a `Boolean` or `Observable<Boolean>`. Let's change it to `showPendingSyncStatus`?